### PR TITLE
Issue #1601 by Inlead: Implement ding_item_cache to DDB CMS.

### PR DESCRIPTION
**Pull request by NicolaeGrigoriu**
http://platform.dandigbib.org/issues/1601

### CODE REVIEW

File: translations/da.po [hunk 1]
```diff
@@ -48241,3 +48241,28 @@ msgstr "Log ud"
...
+msgid "Ding item cache has been cleared"
+msgstr "Materialekarusellen er blevet renset"
+
+#: ding_item_cache.module:15
+msgid "Ding item cache"
+msgstr "Materiale mellemlagring"
+
+#: ding_item_cache.module:16
+msgid "Clear caches of certain modules relying on Ding item caching."
+msgstr ""
```
Comment (guddo) [inline translations/da.po]: "Tømmer mellemlagringen af de moduler som anvender denne cache."

File: translations/da.po [hunk 2]
```diff
@@ -48241,3 +48241,28 @@ msgstr "Log ud"
...
+
+#: ding_item_cache.module:35
+msgid "Select those modules which cache has to be cleared"
+msgstr "Vælg hvilke moduler der skal renses for mellemlagring"
+
+#: ding_item_cache.module:42
+msgid "Clear caches"
+msgstr "Ryd mellemlagring"
+
+
```
Comment (guddo) [inline translations/da.po]: New line too much.

File: translations/da.po [hunk 3]
```diff
@@ -48241,3 +48241,28 @@ msgstr "Log ud"
...
+#: ding_item_cache.module:57
+msgid "Ding item cache has been cleared"
+msgstr "Materialekarusellen er blevet renset"
```
Comment (guddo) [inline translations/da.po]: "Materialekarusellen er blevet renset" -> "Mellemlagringen for materialer er blevet renset"

**Comment by NicolaeGrigoriu** [comment]
Switched to the [283](https://github.com/ding2/ding2/pull/283)

**Pull request closed without merging**